personio-personnel-data-api-oa3.yaml

```diff
@@ -393,11 +393,15 @@ paths:
                       type: string
                       example: '2020-01-31'
                       format: date
-                      description: "Employee hire date. Format: yyyy-mm-dd"
+                      description: "Employee hire date. Format: \"yyyy-mm-dd\". If `status` is not provided, it will be set to `active` if hire date is in the past, or to `onboarding` if it's in the future."
```

Comment:
[nitpick]
```suggestion
                      description: "Employee hire date. Format: \"yyyy-mm-dd\". If `status` is not provided, it will be set to `active` if the hire date is in the past, or to `onboarding` if it's in the future."
```